package.json

```diff
@@ -77,7 +77,8 @@
     "commands": [
       {
         "command": "extension.fabric8AnalyticsWidgetFullStack",
```

Comment:
nit:

Please fix the command name as well.

e.g.
```
fabric8.stackAnalysis
```

Reply:
ack